deployment/README.md

```diff
@@ -0,0 +1,62 @@
+# Deployment
+
+- [AWS Credentials](#aws-credentials)
+- [Terraform](#terraform)
+
+## AWS Credentials
+
+Using the AWS CLI, create an AWS profile named `noaa`:
+
+```bash
+$ aws configure --profile noaa
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-1
+Default output format [None]:
+```
+
+You will be prompted to enter your AWS credentials, along with a default region. These credentials will be used to authenticate calls to the AWS API when using Terraform and the AWS CLI.
+
+## Terraform
+
+First, we need to make sure there is a `terraform.tfvars` file in the project settings bucket on S3. The `.tfvars` file is where we can change specific attributes of the project's infrastructure, not defined in the `variables.tf` file.
+
+Here is an example `terraform.tfvars` for this project:
+
+```hcl
+project     = "Flood Map"
+environment = "Staging"
+aws_region  = "us-east-1"
+
+aws_key_name = "floodmap-stg"
+
+r53_private_hosted_zone = "floodmap.noaa.internal"
```

Comment:
Adding an extra layer of DNS namespacing in the event that we end up hosting more applications here at a later date.